### PR TITLE
Add telemetry event structure

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
Matching https://github.com/Microsoft/vscode-python/pull/2978

### CODE REVIEW

File: src/LanguageServer/Impl/Definitions/ITelemetryService.cs [hunk 1]
```diff
@@ -14,10 +14,17 @@
 // See the Apache Version 2.0 License for specific language governing
 // permissions and limitations under the License.
 
+using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace Microsoft.Python.LanguageServer {
     public interface ITelemetryService {
-        Task SendTelemetry(object o);
+        Task SendTelemetry(TelemetryEvent telemetryEvent);
```
Comment (AlexanderSher) [inline src/LanguageServer/Impl/Definitions/ITelemetryService.cs]: What about implementation?

Comment (MikhailArkhipov) [reply src/LanguageServer/Impl/Definitions/ITelemetryService.cs]: Doh

File: src/LanguageServer/Impl/LanguageServer.cs [hunk 2]
```diff
@@ -99,7 +97,8 @@ private class PublishDiagnosticsParams {
         }
 
         #region Events
-        private void OnTelemetry(object sender, TelemetryEventArgs e) => _telemetry.SendTelemetry(e.value);
+        [Obsolete]
+        private void OnTelemetry(object sender, TelemetryEventArgs e) { }
```
Comment (AlexanderSher) [inline src/LanguageServer/Impl/LanguageServer.cs]: You can simply remove it - it is private.

Comment (MikhailArkhipov) [reply src/LanguageServer/Impl/LanguageServer.cs]: Yep

**Pull request merged**